### ISSUE TITLE
[FR]: feed edit\add redesign

### ISSUE THREAD

**Issue by Owyn** [martinrotter/rssguard#603]
### Brief description of the feature request

![image](https://user-images.githubusercontent.com/1309656/150513019-6c38b5b6-7ef9-436f-9b73-a8d18a0cc01d.png)

(bigger fields are nice but ok\cancel buttons are unavailable now...and [x] is disabled )

Suggestions about redesigning this:
- Type and Encoding are useless for users and just confuse them, in other readers it's not something you can even choose ignoring auto-detection (and even if you could - you'd only break things)

= put type and encoding to the right of URL (and make url drop-down little smaller in width) so these three would be in one line, but only after the URL is fetched (when these two are auto-detected)

**Comment by martinrotter** [comment]
> Type and Encoding are useless for users and just confuse them

Of course they are useful. I agree that in majority of cases, they are detected properly, but sometimes, they are not.

But redesign, yes, thinking about it. There is too many items on single page, indeed.

**Comment by martinrotter** [comment]
I will probably create two tabs from general tab:

"Source" - this tab will include URL/script textboxes, type and encoding and also "Fetch metadata button".
"Metadata" - this tab will include title/description, icon, category, etc

must think about it

**Comment by Owyn** [comment]
> script textboxes

would be nice if those were minimized to fit the text when not in focus, and expanded when focused

also would be more than welcomed "recently used scripts" dropdown near it so we could just use last used script with its params

also, "what is script?" thingy could be done the usual way - when user hovers over the text a question mark appears and after a sec a tooltip pops up with an explanation

> tabs

personally I hope it won't require more clicking around and searching things, in most RSS clients you just have to enter feed url and press OK and that's it...

todays monitors are (ultra-) wide format - just make window wider with more stuff, not taller... and show only currently actual stuff, rest should be minimized \ greyed out (I think minimizing stuff is better than making multiple tabs) - e.g. a line saying "http auth required? [ ]" - you click it - 4 more lines appear etc

**Comment by martinrotter** [comment]
yes

**Comment by martinrotter** [comment]
OK, i did some minor re-design to cope at least with some cases, dialog is now bit smaller. Textboxes for URL and post-process are stretched, encoding box is moved after type box etc.

![image](https://user-images.githubusercontent.com/1255302/151672034-456978ca-3cb3-499a-b533-92a2dbb29d1c.png)

**Comment by Owyn** [comment]
I still hope we could get "recently used script" string drop-down menu so we could reuse used before values for (post-proccess)-script input

**Comment by martinrotter** [comment]
Separate ticket for that pls.
